Calculator_program.c

```diff
@@ -0,0 +1,127 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <ctype.h>
+
+#define MAX_LEN 100 // Maximum length
+
+// Function to calculate expressions 
+float calculateExpression(char *expr) {

```

Comment:
Why have you written everything in one function? This function can be divided into small functions which do one specific task. Practice writing functions in such a way that one function does one task.

Reply:
👍